lib/hanami/cli/generators/gem/app/app_js.erb

```diff
@@ -0,0 +1 @@
+import "../stylesheets/app.css";
```

Comment:
Forgive me if this is an ignorant question... but why do we need to import the app's _stylesheet_ from inside this JavaScript file?

Reply:
@timriley This is because Esbuild is only processing assets (JS and CSS) referenced from our entry points.

Entry points are:
* `app/assets/javascripts/**/app.js`
* `slices/*/assets/javascripts/**/app.js`

We support many other extensions: TS, MJS, JSX, etc...

Esbuild does that for efficiency reasons: only process (and bundle) the required code.

---

We have two options:

1. (Current) Take Esbuild defaults. This means that CSS needs to be referenced directly (or indirectly) from entry points. That's why we import the CSS from that file.
2. Implement a custom strategy for CSS, just always process ALL the stylesheets ALL the time. Like we do for static assets.

Reply:
Thanks for clarifying this for me, @jodosha!

I think that following Esbuild's defaults is probably the most sensible option for us now. Let's roll with this and ship it with the next beta, then decide after that whether we might need to directly process CSS files?

Reply:
To help with our eventual decision here, it might be good to come up with a range of different "personas" for asset structures (e.g. a simple site with CSS only but no JS, a site with React and Tailwind, and maybe some third structure) — then we can survey how well we serve each scenario with our current approach vs an approach with direct CSS handling. What do you reckon?